app/src/main/res/values/strings.xml

```diff
@@ -178,7 +178,7 @@
     <string name="sort_ascending">Lowest first ↓</string>
     <string name="sort_groups_before">Groups before</string>
     <string name="sort_recycle_bin_bottom">Recycle bin at the bottom</string>
-    <string name="sort_db">DB sort order</string>
+    <string name="sort_db">Database Sorting Order</string>
```

Comment:
It's like sort_title or sort_username but not yet implemented.
Must be something like **Natural Database**

Reply:
What is natural database?